### PR TITLE
Update to MXNet 1.1.0

### PR DESCRIPTION
**Pull request by fhieber**
Changed Sequence{Mask,Last} operators to use the new axis argument to avoid a bunch of transposes.

#### Pull Request Checklist ##
- [x] Changes are complete (if posting work-in-progress code, prefix your pull request title with '[WIP]'
until you can check this box.
- [x] Unit tests pass (`pytest`)
- [x] Were system tests modified? If so did you run these at least 5 times to account for the variation across runs?
- [x] System tests pass (`pytest test/system`)
- [x] Passed code style checking (`./pre-commit.sh` or manual run of pylint & mypy)
- [x] You have considered writing a test
- [x] Updated major/minor version in `sockeye/__init__.py`. Major version bump if this is a backwards incompatible change.
- [x] Updated CHANGELOG.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by davvil** [comment]
Why is it backwards incompatible, though? Did the format of files change with the new mxnet?

**Comment by fhieber** [comment]
A good point, this just changes the comp. graph but nothing in the parameters so this is actually not a breaking change!

File: test/unit/test_utils.py [hunk 1]
```diff
@@ -201,6 +202,16 @@ def test_check_version_checks_major():
     assert "Given major version (%s) does not match major code version (%s)" % (version, __version__) == str(e.value)
 
 
+def test_version_matches_changelog():
+    """
+    Tests whether the last version mentioned in CHANGELOG.md matches the sockeye version (sockeye/__init__.py).
+    """
+    pattern = re.compile(r'''## \[([0-9.]+)\]''')
```
Comment (davvil) [inline test/unit/test_utils.py]: I would add '^' and '$' to be safer.

**Comment by davvil** [comment]
Approval still holds after changes.

**Pull request merged**